proto/vfs.proto

```diff
@@ -0,0 +1,172 @@
+syntax = "proto3";
```

Comment:
Could you add a comment at the top linking the source that this is from / based on?

Reply:
Done.